fvm/transaction.go

```diff
@@ -101,7 +101,7 @@ func (proc *TransactionProcedure) MemoryLimit(ctx Context) uint64 {
 	// TODO for BFT (enforce max computation limit, already checked by collection nodes)
 	// TODO let user select a lower limit for memory (when its part of fees)
 
-	memoryLimit := uint64(DefaultMemoryLimit) // TODO use the one set by tx
+	memoryLimit := ctx.MemoryLimit // TODO use the one set by tx
```

Comment:
```suggestion
	memoryLimit := ctx.MemoryLimit
```

Reply:
hm.... I think that still applies. Let me go check.

Reply:
yes, this todo still holds.